File: http/src/main/java/webconnect/com/webconnect/listener/AnalyticsListener.java
package webconnect.com.webconnect.listener;

/**
 * Created by clickapps on 18/1/18.
 */

public interface AnalyticsListener {
    void onReceived(long timeTakenInMillis, long bytesSent, long bytesReceived, boolean isFromCache);
}

File: http/src/main/java/webconnect/com/webconnect/listener/AnalyticsListener.kt
package webconnect.com.webconnect.listener

/**
 * Created by clickapps on 18/1/18.
 */

interface AnalyticsListener {
    fun onReceived(timeTakenInMillis: Long, bytesSent: Long, bytesReceived: Long, isFromCache: Boolean)
}
